FidoCore.md

```diff
@@ -1384,3 +1475,154 @@ begin
   Slots.Register(FSlots, FViewModel, Self)
 end;
 ```
+
+Slots can be of two types:
+
+- `stSynched`, the slot will be synchronized with the main thread  
+- `stNotSynched`, the slot will not be synchronized with the main thread.
+
+## Events driven architecture
+
+Units folder: `EventsDriven`.
+
+![](C:\Users\mendrix\Documents\fidolib - github\diagrams\Events driven.svg)
```

Comment:
This ain't gonna show up chief

Reply:
That's what happens when you are lazy and use the wysiwyg editors.
I'll do a commit.